src/components/Status/Status.js

```diff
@@ -35,29 +41,64 @@ OverlayStatus.defaultProps = {
 
 OverlayStatus.propTypes = {
   icon: PropTypes.string,
-  text: PropTypes.string.isRequired,
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
   overlay: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
 };
 
-export const LinkStatus = ({ icon, children, linkMessage, linkTo }) =>
-  linkTo ? (
-    <Status icon={icon}>
-      <Link to={linkTo} title={linkMessage}>
-        {children}
-      </Link>
-    </Status>
+export const LinkStatus = ({ icon, children, title, information, linkMessage, linkTo, message }) => {
```

Comment:
The LinkStatus needs to be more expressive. You should test this with multiple links and progresses.

I would suggest API like this 

```jsx
<LinkStatus icon={icon} title={title} message={message} >
  <StatusField title={title}>  {/*  optional - name of the pod? */}
    <Progress progres={progress} />  {/* this can  be progress bar in the future */}
    <Link linkTo={linkTo} /> {/*  link to the pod? */}
  </StatusField>
  <StatusField>
    <Link />
  </StatusField>
</LinkStatus>
```

Reply:
thoughts?

Reply:
Nice :exclamation:  I like it